docs/architecture.md

```diff
@@ -75,3 +116,114 @@ the decision.
 ### Attestation diagram
 
 ![attestation](images/Fobnail-flows-attestation.png)
+
+---
+
+## Keys and certificates
+
+### Platform Owner
+
+#### Knows in advance
+
+- Platform Owner key (signing and encryption)
+  - Fobnail and remote platform provisioning
+  - used for TLS and for signing certificates
+- Platform Owner certificate
+  - Fobnail and remote platform provisioning
+  - has to be able to provide whole CA chain
+  - both Fobnail Token and Platform must trust root CA in this chain
+- root CA certificate for EK certificate chain
+  - remote platform provisioning
+
+#### Receives
+
+- Fobnail Identity key
+  - Fobnail provisioning
+- Fobnail Encryption key
```

Comment:
Encryption key for internal storage? Shouldn't it be known only by Fobnail?

Reply:
I agree, certificate doesn't make sense in that case. I left this and similar inconsistencies in https://github.com/fobnail/docs/issues/20

Reply:
Can we link to this issue in the document in this place then?